### ISSUE TITLE
Search input in community sidebar

### ISSUE THREAD

**Issue by KaKi87** [Xyphyn/photon#295]
**Is your feature request related to a problem? Please describe.**
Search is global by default, and when wanting to search in a community while currently browsing it, clicking *Search* requires clicking an arrow button then inputting the community.

**Describe the solution you'd like**
A search input in the sidebar would allow searching directly within the current community.

**Describe alternatives you've considered**
Automatically inputting the community in the search page while clicking a newly added *Search* button, but directly adding an input would spare a click.

**Additional context**
None

---

Thanks

**Comment by Xyphyn** [comment]
Duplicate of #290 in idea

**Comment by KaKi87** [comment]
No, theirs is about searching communities, mine is about searching posts within a community.

**Comment by Xyphyn** [comment]
Oh NVM then

**Comment by KaKi87** [comment]
Better late than never ! I've been using [Tesseract](https://github.com/asimons04/Tesseract) in big part for that feature.
